config/locales/en.yml

```diff
@@ -443,7 +443,7 @@ en:
           above_hard_max: "Rent is higher than the absolute maximum expected for a property of this type based on this period."
       charges:
         complete_1_of_3: "Answer either the ‘household rent and charges’ question or ‘is this accommodation a care home‘, or select ‘no’ for ‘does the household pay rent or charges for the accommodation?’"
-        missing_charges: "Please enter the %{question}. If there is no %{question}, please enter '0'."
+        missing_charges: "Please enter the %{sentence_fragment}. If there is no %{sentence_fragment}, please enter '0'."
```

Comment:
The idea behind this is to always assume if the expected variable is "question" it will include it's own punctuation. And for any other inputs like this were they are put in a pre formatted sentence to call them "sentence_fragment".